store/views.py

```diff
@@ -77,6 +107,52 @@ def loanBookView(request):
 @csrf_exempt
 @login_required
 def returnBookView(request):
-    pass
+    response_data = {
+        'message': "failure"
+    }
+    data = request.POST
+    copybook_id = data.get('bid')
+    if copybook_id is not None:
+        copybook = BookCopy.objects.get(id=copybook_id)
+        copybook.borrow_date = None
+        copybook.borrower_id = None
+        copybook.status = False
+        copybook.save()
```

Comment:
There must be a validation in the backend when a user is returning the book, to make sure that he has only borrowed the book. Otherwise, a simple POST request will make the BookCopy to be returned, and would set its status as True.